pkg/terraform/store.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/spf13/afero"
 	"k8s.io/apimachinery/pkg/types"
+	k8sExec "k8s.io/utils/exec"
```

Comment:
```suggestion
	kexec "k8s.io/utils/exec"
```

We usually go for prefixing with `k` if there is a counterpart, like `kerrors` for k8s errors package.

Reply:
In fact, it seems there is one last usage of `os/exec` in this file in line 148 where we call `terraform init`, so if we remove that, we can just use `exec`.

Reply:
Done!